src/server/service/diagnostics/sys.rs

```diff
@@ -514,9 +535,9 @@ mod tests {
         let prev_io = ioload::IoLoad::snapshot();
         let mut collector = vec![];
         load_info((prev_cpu, prev_nic, prev_io), &mut collector);
-        #[cfg(linux)]
+        #[cfg(target_os = "linux")]
```

Comment:
I'm new to this change, what's the difference?

Reply:
https://github.com/rust-lang/rust-clippy/issues/3949

Reply:
There is never a cfg called `linux`. So `#[cfg(linux)]` just remove the code in all cases.

Reply:
There is `#[cfg(unix)]` but actually we rely on Linux-specific features, so we should use `#[cfg(target_os = "linux")]`